tests/integration/components/sl-radio-group-test.js

```diff
@@ -0,0 +1,169 @@
+    );
+});
+
+test( 'The readonly state applies to sl-radio children', function( assert ) {
+    this.render( hbs`
+        {{#sl-radio-group readonly=true name="testName"}}
+            {{#sl-radio label="One" value="one"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Two" value="two"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Three" value="three"}}
+            {{/sl-radio}}
+        {{/sl-radio-group}}
+    ` );
+
+    assert.equal(
+        this.$( 'input[readonly]' ).length,
+        3,
+        'Rendered component has three readonly inputs'
+    );
+});
+
+test( "Inline true sets sl-radio children's inline property to true", function( assert ) {
+    this.render( hbs`
+        {{#sl-radio-group inline=true name="testName"}}
+            {{#sl-radio label="One" value="one"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Two" value="two"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Three" value="three"}}
+            {{/sl-radio}}
+        {{/sl-radio-group}}
+    ` );
+
+    assert.equal(
+        this.$( '.sl-radio.radio' ).length,
+        0,
+        'Rendered component children buttons are not inline'
+    );
+
+    assert.equal(
+        this.$( '.sl-radio.radio-inline' ).length,
+        3,
+        'Rendered component children buttons are inline'
+    );
+});
+
+test( "Inline false sets sl-radio children's inline property to false", function( assert ) {
+    this.render( hbs`
+        {{#sl-radio-group inline=false name="testName"}}
+            {{#sl-radio label="One" value="one"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Two" value="two"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Three" value="three"}}
+            {{/sl-radio}}
+        {{/sl-radio-group}}
+    ` );
+
+    assert.equal(
+        this.$( '.sl-radio.radio-inline' ).length,
+        0,
+        'Rendered component has zero inline radio buttons'
+    );
+
+    assert.equal(
+        this.$( '.sl-radio.radio' ).length,
+        3,
+        'Rendered component has three default (non-inline) radio buttons'
+    );
+});
+
+test( 'Value changes when sl-radio child selected', function( assert ) {
+    this.set( 'value', 'jeremy' );
+    this.render( hbs`
+        {{#sl-radio-group value=value name="testName"}}
+            {{#sl-radio label="One" value="eric"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Two" value="josh"}}
+            {{/sl-radio}}
+            {{#sl-radio label="Three" value="michael"}}
+            {{/sl-radio}}
+        {{/sl-radio-group}}
+    ` );
+
+    Ember.run( () => {
+        const $radioButton = this.$( 'input[value="eric"]' );
+        $radioButton.click();
+    });
+
+    assert.equal(
+        this.get( 'value' ),
+        'eric',
+        '"eric" value is selected'
+    );
+
+    Ember.run( () => {
+        const $radioButton = this.$( 'input[value="josh"]' );
+        $radioButton.click();
+    });
+
+    assert.equal(
```

Comment:
This second assertion is not necessary, as the first one covers this behavior just fine